.evergreen/download-mongodb.sh

```diff
@@ -494,8 +494,7 @@ get_mongodb_download_url_for ()
          if [ -z $MONGODB_60 ]; then
            VERSION_INCLUDES_CRYPT_SHARED=NO
          fi ;;
-      rapid) MONGODB_DOWNLOAD_URL=$MONGODB_RAPID
-         VERSION_INCLUDES_CRYPT_SHARED=NO ;;
+      rapid) MONGODB_DOWNLOAD_URL=$MONGODB_RAPID ;;
```

Comment:
I believe this was missed when `VERSION_RAPID` was updated to 6.1 in #237. The previous rapid version (5.3) didn't include crypt_shared, and I believe that's why this was included. Since the rapid release will now always include crypt_shared, I believe this should be safe to remove. Any objections to this @kevinAlbs?

Reply:
SGTM. Thank you.